pkg/network/protocols/amqp/testdata/docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '3'
```

Comment:
TODO: Add the docker compose to system-probe.py once https://github.com/DataDog/datadog-agent/pull/14589 is merged